src/utils/connectionStringUtil.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AuthenticationTypes } from '../models/interfaces';
+
+export function getConnectionString(server: string, authType: string, database: string, userName: string, password: string): string {
```

Comment:
Why not use the STS functionality instead? This isn't very future-proof and will miss out on any custom settings the user might have specified for their connection.

Reply:
as said in the PR description. This would require the same functionality that we discussed in STS of accepting connectionInfo otherwise would have to reconnect to the Server in order to get the proper uri to send to the connection string request. Will address this again since most work seems to be going to STS in near future.